xbmc/cores/RetroPlayer/RetroPlayerAutoSave.cpp

```diff
@@ -47,7 +48,7 @@ void CRetroPlayerAutoSave::Process()
     if (m_bStop)
       break;
 
-    if (m_player->GetSpeed() > 0.0f)
+    if (CDataCacheCore::GetInstance().GetSpeed() > 0.0f)
```

Comment:
@FernetMenta No problem, RetroPlayer has its own internal playback API. I've moved over to it in #12493